rootfs/etc/cont-init.d/04-svc-main.sh

```diff
@@ -82,7 +82,7 @@ lnms migrate --force --no-ansi --no-interaction
 artisan db:seed --force --no-ansi --no-interaction
 
 echo "Clear cache"
-artisan cache:clear --no-interaction
+artisan cache:clear --no-interaction || true
```

Comment:
```suggestion
artisan cache:clear --no-interaction || (echo "WARNING: There was some issue while trying to clear cache. This may cause problems while running, check the logs for more details."; true)
```